### ISSUE TITLE
Search function

### ISSUE THREAD

**Issue by superuser-does** [wbrawner/SimpleMarkdown#59]
I keep all my personal notes in Markdown and sync them from PC to a folder on my phone. As I've written most of them on my computer, you can imagine length is not an issue. ;)

A simple search function would be very handy for my use.

**Comment by wbrawner** [comment]
Hey, thanks for the suggestion. I'll look into it when I have a moment :)

**Comment by jpbourgeon** [comment]
I upvote this feature
The editor is ready good. Thank you for this app.

Being able to search something inside a long file would be a treat.

**Comment by KeronCyst** [comment]
Agreed, I was editing a 3k-line-long doc and that was painful to the point of wanting to switch to something else. When will Ctrl+F get added (I type from a keyboard)?
